worldedit-core/src/main/java/com/sk89q/worldedit/command/ToolCommands.java

```diff
@@ -265,7 +261,6 @@ public void deltree(Player player, LocalSession session) throws WorldEditExcepti
 
     @Command(
         name = "farwand",
-        aliases = { "/warwand" },
```

Comment:
That is a reasonable remove, because the alias is just trash.

Reply:
I mean that's probably a typo...